auto_nag/bzcleaner.py

```diff
@@ -355,9 +355,11 @@ def set_needinfo(self, dryrun):
                 res[bugid] = data
         return res
 
-    def has_individual_autofix(self):
-        """Check if the autofix is the same for all bugs or different for each bug"""
-        return False
+    def guess_how_autofix(self, changes):
+        # check if we have a dictionary with bug numbers as keys
+        # return True if all the keys are bug number
+        # (which means that each bug has its own autofix)
+        return all(k.isdigit() for k in changes.keys())
```

Comment:
It'd be nice to allow integers too, so the scripts don't have to convert the integer IDs into strings.